planner/core/rule_join_elimination.go

```
@@ -74,7 +75,8 @@ func (o *outerJoinEliminator) isAggColsAllFromOuterTable(outerPlan LogicalPlan,
 	}
 	for _, col := range aggCols {
 		columnName := &ast.ColumnName{Schema: col.DBName, Table: col.TblName, Name: col.ColName}
-		if c, _ := outerPlan.Schema().FindColumn(columnName); c == nil {
+		if c, err := outerPlan.Schema().FindColumn(columnName); c == nil {
+			log.Debug(err)
```

I think debug level is too low.

---

This error could be ignored @eurekaka

---

This log is too simple.

---

Done